dashboard/test/ui/README.md

```diff
@@ -8,17 +8,18 @@ Automated UI tests for the dashboard.
 
 This is the best option for rapid iteration while writing a new test. ChromeDriver will run your tests in a new window on your machine.
 
-1. [Download the chromedriver](https://github.com/SeleniumHQ/selenium/wiki/ChromeDriver)
-2. Start the chromedriver in the background (or a separate window): `/path/to/your/downloaded/chromedriver &`
-3. `cd` to the directory of this README.
-4. `bundle install`
-5. `rbenv rehash`
-6. `./runner.rb -l`
+1. `cd` to the directory of this README.
+2. if this is your first time running tests via chromedriver:
+  a. run `bundle install` and then `rbenv rehash`
+  b. confirm that `which chromedriver` returns `~/.rbenv/shims/chromedriver` 
```

Comment:
This might be different on different platforms or with different ruby configurations:
```
╰○ which chromedriver
/home/brad/.rvm/gems/ruby-2.2.3/bin/chromedriver
```
This is after following the `chromedriver-helper` instructions for updating chromedriver, and confirming the version changed, so I'm pretty sure mine is managed there too.  Maybe sufficient to confirm that chromedriver is found after these steps.